### ISSUE TITLE
Add a landing page listing all Dar documents/projects in the archive folder

### ISSUE THREAD

**Issue by nuest** [minrk/jupyter-dar#6]
AFAIC the `dar-server` does not have a "list me all Dar projects in the storage" function, but something like that would be needed for an index page.

@nokome: Does some other component provide this functionality?

**Comment by nokome** [comment]
Good point. I'm not too familiar with `dar-server` that's a question for @michael or @oliver----

**Comment by obuchtala** [comment]
@nuest There is nothing like this yet. You can use html params for that.
See for example https://github.com/stencila/stencila/blob/develop/app/index.html
`darServer` has some arguments to control which folder is used as root folder for archives: https://github.com/substance/dar-server/blob/master/src/serve.js#L10-L15

**Comment by nuest** [comment]
@oliver---- Thanks for clarifying. HTML params require us to know the name of the Dar directory beforehand, which we do not. Maybe to clarify what we want:

_A user has one (or more) Dar directories in a repository on GitHub. When she opens the repo with binder, we would like to open Stencila with either the only Dar, or show a list of available ones._

As of now, the user has to provide the directory name. It would be cool if she does not have to.

(Also, we assume that a Dar will always be a directory. Will read the spec later today ;-) )

**Comment by obuchtala** [comment]
(moved to gitter for more discussions)

**Comment by nuest** [comment]
@oliver---- @nokome @michael I think it would be great if we can "wrap up" the integration done at eLife Sprint during the next couple of weeks. _Do you think this issue (a landing page) should be included?_ I don't know how it relates to the other ideas for integrations mentioned on Gitter, so your input would be useful.

@oliver---- AFAIR you made some change to dar-server supporting this - are those already released?

**Comment by nokome** [comment]
I think the conclusion was just to keep it a single Dar for now i.e. no landing page at least for now.

**Comment by michael** [comment]
Would stick with a single Dar for now too. The listArchives API is there in DarServer, but no need to use it.
